### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by zerodarkzone** [namkazt/LightShadowSystem#2]
Hi,
First of all, this looks great, I compiled this code with cocos and it looks quite good. Are you going to keep working on it? the shader code works and it seems the project only lacks a real interface to use it.
I hope you finish it one day.

**Comment by namkazt** [comment]
yeah. it have performance problem also i do not have time to fix it. im not working with cocos2dx any more so just let it here.
